content/applications/general/payment_acquirers/adyen.rst

```diff
@@ -99,5 +99,27 @@ where payments will be made (the URLs of the servers hosting your Odoo instances
    :align: center
    :alt: Allows payments originated from a specific domain.
 
+Capture payments after the delivery
+-----------------------------------
+
+Adyen allows you to capture an amount manually instead of having an immediate capture.
+
+To set it up, enable the **Capture Amount Manually** on Odoo, as explained in the :ref:`payment
+acquirers documentation <payment_acquirers/capture_amount>`, then on Adyen.
+
+To do so, open your Adyen Merchant Account, go to :menuselection:`Account --> Settings`, and set the
+**Capture Delay** to **manual**.
+
+.. image:: media/adyen_capture_delay.png
+   :align: center
+   :alt: Capture Delay settings in Adyen
+
+.. warning::
+   * If you set the **Capture Amount Manually** on Odoo but not on Adyen, Odoo can't detect it.
+     Therefore, your authorization request would be automatically approved in an immediate capture
+     on Adyen's side.
+   * After **7 days**, if the authorization hasn't yet been captured, the customer has the right to
+     **revoke** it.
```

Comment:
Like, for real? I thought it was automatic for all providers. How would the customer do that from their side?

Reply:
They call their bank, send a request from their bank app... Actually, this may be better if we could put it on the `payment_acquirers` page, if we can confirm the delay is the same for all acquirers. But yes, I can definitely do so from my bank app.
Thing is, the provider Stripe (or Authorized?), for instance, specify that an authorization is automatically revoked after 30 days if it's not captured/void yet, Adyen don't specify this, but instead speaks of the 7 days for the client, which the other provider don't mention.
I think the actual delay(s ?) might depend more on your bank than on the provider, but I'm not sure. Either way, I feel this should be written somewhere.

Reply:
omg, I didn't test this notification in ma task... Is it a normal void notification? I hope it's a normal void notification... :fearful:

Reply:
I think it's fine to only mention it in Adyen doc for now, especially if it depends on the provider.

However, you capture a payment (or a transaction), not an authorization.